youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,588 @@
+msgstr "הורדת youtube-dl נכשלה [{0}]"
+
+#: youtube_dl_gui/mainframe.py:132
+msgid "Successfully downloaded youtube-dl"
+msgstr "youtube-dl הורד בהצלחה"
+
+#: youtube_dl_gui/mainframe.py:134
+msgid "Unable to open directory: '{dir}'. The specified path does not exist"
+msgstr "פתיחת התיקייה נכשלה: '{dir}'. הנתיב שצוין אינו קיים"
+
+#: youtube_dl_gui/mainframe.py:136
+msgid "Error while shutting down. Make sure you typed the correct password"
+msgstr "שגיאה בעת כיבוי. ודא שהקלדת את הסיסמה הנכונה"
+
+#: youtube_dl_gui/mainframe.py:138
+msgid "Shutting down system"
+msgstr "כיבוי המערכת"
+
+#: youtube_dl_gui/mainframe.py:141
+msgid "Extension"
+msgstr "סיומת"
+
+#: youtube_dl_gui/mainframe.py:142
+msgid "Size"
+msgstr "גודל"
+
+#: youtube_dl_gui/mainframe.py:143
+msgid "Percent"
+msgstr "אחוז"
+
+#: youtube_dl_gui/mainframe.py:144
+msgid "ETA"
+msgstr ""
+
+#: youtube_dl_gui/mainframe.py:145
+msgid "Speed"
+msgstr "מהירות"
+
+#: youtube_dl_gui/mainframe.py:146
+msgid "Status"
+msgstr "מצב"
+
+#: youtube_dl_gui/mainframe.py:235
+msgid "Get URL"
+msgstr "קבל כתובת אתר"
+
+#: youtube_dl_gui/mainframe.py:236
+msgid "Get command"
+msgstr "קבל פקודה"
+
+#: youtube_dl_gui/mainframe.py:237
+msgid "Open destination"
+msgstr "פתח יעד"
+
+#: youtube_dl_gui/mainframe.py:238
+msgid "Re-enter"
+msgstr "הזן מחדש"
+
+#: youtube_dl_gui/mainframe.py:458
+msgid "Resume"
+msgstr "התחל מחדש"
+
+#: youtube_dl_gui/mainframe.py:480
+msgid "Video"
+msgstr "וידיאו"
+
+#: youtube_dl_gui/mainframe.py:484
+msgid "Audio"
+msgstr "שמע"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "No items selected. Please pick an action"
+msgstr "לא נבחרו פריטים. נא בחר פעולה"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "Remove all"
+msgstr "מחק הכל"
+
+#: youtube_dl_gui/mainframe.py:516
+msgid "Remove completed"
+msgstr ""
+
+#: youtube_dl_gui/mainframe.py:534
+msgid "Are you sure you want to remove selected items?"
+msgstr "האם אתה בטוח שברצונך להסיר פריטים נבחרים?"
+
+#: youtube_dl_gui/mainframe.py:546
+msgid "Item is active, cannot remove"
+msgstr "הפריט פעיל, לא ניתן להסיר"
+
+#: youtube_dl_gui/mainframe.py:579
+msgid "Item is not completed"
+msgstr "הפריט לא הושלם"
+
+#: youtube_dl_gui/mainframe.py:668
+msgid "Update in progress. Please wait for the update to complete"
+msgstr "עדכון בתהליך. המתן עד להשלמת העדכון"
+
+#: youtube_dl_gui/mainframe.py:716
+msgid "Logging is disabled"
+msgstr "רישום מושבת"
+
+#: youtube_dl_gui/mainframe.py:891
+msgid "Shutdown"
+msgstr "כיבוי"
+
+#: youtube_dl_gui/mainframe.py:891
+msgid "Shutting down in {0} second(s)"
+msgstr "כיבוי בעוד {0} שניות"
+
+#: youtube_dl_gui/mainframe.py:980
+msgid "No items to download"
+msgstr "אין פריטים להורדה"
+
+#: youtube_dl_gui/mainframe.py:1060
+msgid "Are you sure you want to exit?"
+msgstr "אתה בטוח שאתה רוצה לצאת?"
+
+#: youtube_dl_gui/mainframe.py:1060
+msgid "Exit"
+msgstr "יציאה"
+
+#: youtube_dl_gui/mainframe.py:1301 youtube_dl_gui/mainframe.py:1451
+msgid "Cancel"
+msgstr "בטל"
```

Comment:
This translates to *void, null, idle* which have nothing in common with the *msgid* field.
Are you sure this is correct?

Reply:
This is for sure the translation yoy can check at
https://translate.google.com/#en/iw/cancel
As you can see it translate to לבטל but בטל  is also